examples/flags/README.md

```diff
@@ -28,6 +28,7 @@ There are several supported flags (for more accurate list, see package `pkg/flag
 * `features`
 * `labels`
 * `kubeconfig`
+* `kube-context`
```

Comment:
@maruina one more minor suggestion
Let's call the flag just `context`. This is to stay aligned with the naming convention (usually) followed by other tools in the kubernetes ecosystem.

Reply:
Yep, done. That's why I was suggesting it in the fist place